lib/visitor.js

```diff
@@ -6,11 +6,7 @@ const underscoreCode = "_".charCodeAt(0);
 // This Visitor API was inspired by a similar API provided by ast-types:
 // https://github.com/benjamn/ast-types/blob/master/lib/path-visitor.js
 
-function Visitor() {
-  this.visit = this.visit.bind(this);
-  this.visitWithoutReset = this.visitWithoutReset.bind(this);
-  this.visitChildren = this.visitChildren.bind(this);
-}
```

Comment:
Let's keep this binding so that we don't have to keep creating arrow functions every time we call one of these methods.